src/DssSpell.sol

```diff
@@ -49,100 +51,33 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
-        DssExecLib.setContract(MCD_END, "dog", MCD_DOG);
-        DssExecLib.authorize(MCD_DOG, MCD_END);
-        DssExecLib.deauthorize(MCD_DOG, MCD_END_OLD);
-
-        require(MCD_SPOT == EndAbstract(MCD_END_OLD).spot(), "sanity-error-spot");
-        DssExecLib.setContract(MCD_END, "spot", MCD_SPOT);
-        DssExecLib.authorize(MCD_SPOT, MCD_END);
-        DssExecLib.deauthorize(MCD_SPOT, MCD_END_OLD);
-
-        require(MCD_POT == EndAbstract(MCD_END_OLD).pot(), "sanity-error-pot");
-        DssExecLib.setContract(MCD_END, "pot", MCD_POT);
-        DssExecLib.authorize(MCD_POT, MCD_END);
-        DssExecLib.deauthorize(MCD_POT, MCD_END_OLD);
-
-        DssExecLib.setContract(MCD_ESM, "end", MCD_END);
-        DssExecLib.authorize(MCD_END, MCD_ESM);
-        DssExecLib.deauthorize(MCD_END_OLD, MCD_ESM);
-
-        bytes32[] memory ilks = IlkRegistryAbstract(ILK_REGISTRY).list();
-        for (uint256 i = 0; i < ilks.length; i++) {
-            bytes32 ilk = ilks[i];
-
-            address xlip = IlkRegistryAbstract(ILK_REGISTRY).xlip(ilk);
-            if (IlkRegistryAbstract(ILK_REGISTRY).class(ilks[i]) < 3) {
-                DssExecLib.deauthorize(xlip, MCD_END_OLD);
-                DssExecLib.authorize(xlip, MCD_END);
-            }
-
-            try DssExecLib.removeReaderFromWhitelist(IlkRegistryAbstract(ILK_REGISTRY).pip(ilk), MCD_END_OLD) {} catch {}
-            try DssExecLib.addReaderToWhitelist(IlkRegistryAbstract(ILK_REGISTRY).pip(ilk), MCD_END) {} catch {}
-        }
-
-        // --------------------------- New DssFlash ----------------------------
-
-        DssExecLib.authorize(MCD_VAT, MCD_FLASH);
-        uint256 newMax = FlashAbstract(MCD_FLASH_LEGACY).max() / 2;
-        DssExecLib.setValue(MCD_FLASH, "max", newMax);
-        DssExecLib.setValue(MCD_FLASH_LEGACY, "max", newMax);
-
-        // ------------------------ Transfer MKR to SES ------------------------
-        DSTokenAbstract(MCD_GOV).transfer(SES_WALLET, 541.47 ether);
-
-        // ----------------------------- Chainlog ------------------------------
-
-        DssExecLib.setChangelogAddress("MCD_END", MCD_END);
-        DssExecLib.setChangelogAddress("MCD_CURE", MCD_CURE);
-        DssExecLib.setChangelogAddress("MCD_FLASH", MCD_FLASH);
-        DssExecLib.setChangelogAddress("MCD_FLASH_LEGACY", MCD_FLASH_LEGACY);
-        DssExecLib.setChangelogVersion("1.13.0");
+        // ---------------------------- Lid for Flap ---------------------------
+        DssExecLib.setValue(MCD_FLAP, "lid", 30_000 * RAD);
+
+        // ------------------------------ ESM Min ------------------------------
+        DssExecLib.setValue(MCD_ESM, "min", 150_000 * WAD);
+
+        // ---------------------------- Transfer MKR ---------------------------
+        DSTokenAbstract(MCD_GOV).transfer(SIDESTREAM_WALLET, 243.7953 ether);
+        DSTokenAbstract(MCD_GOV).transfer(DUX_WALLET,        355.86   ether);
```

Comment:
The proposal for this lists the MKR amount at 366.35 MKR. Is there a place where this amount was modified?

@LongForWisdom 

https://forum.makerdao.com/t/mip40c3-sp27-development-ux-core-unit-mkr-budget-dux-001/9777

![Screenshot from 2022-05-24 12-21-00](https://user-images.githubusercontent.com/2374718/170084685-31eec82e-88b1-4bd2-be4a-8d53df18d85a.png)

Reply:
Looks like it's coming from here: 

https://github.com/makerdao-dux/transparency-reporting/blob/main/Monthy%20Budget%20Statements/2022-04.md#3-mkr-vesting-overview

Let's add a comments above each of these pointing to the correct sources.

Reply:
added: https://github.com/makerdao/spells-mainnet/pull/247/commits/f6f02350ec1a220480c4593c10b1e59e168ff69b

Reply:
> Let's add a comments above each of these pointing to the correct sources.

We should standardize across the team if we are going to do this or not: https://github.com/makerdao/spells-mainnet/pull/246#discussion_r874206667

Reply:
I've been doing it. GovAlpha provides the links in the spreadsheet and they're useful for historical purposes.